manual/de/Help_ch_Tools_Assistants.xml

```diff
@@ -5190,121 +5190,491 @@
-          <emphasis>US Dollars</emphasis> or <emphasis>IBM Common Stock</emphasis>.
-        </para>
-      </listitem>
+        <glossdef>
+          <para>Möchten Sie ein Wertpapier oder eine Währung bearbeiten, markieren Sie es in der Liste, und die
+            Daten werden in dem Dialog <guilabel>Wertpapier bearbeiten</guilabel> bzw.
+            <guilabel>Währung bearbeiten</guilabel> zur Änderungen angeboten.
+          </para>
+        </glossdef>
+      </glossentry>
 
-      <listitem>
-        <para><emphasis role="bold">Code:</emphasis> This is any numeric or alphanumeric code that is used to
-          identify the commodity. The <acronym>ISIN</acronym> code is an international unique identifying numeric
-          string that is associated with every stock, bond or mutual fund, and most kinds of traded options,
-          futures and commodities. There are also regional valid codes available like <acronym>CUSIP</acronym> in
-          the North America, <acronym>WKN</acronym> in DE, … This code is not required.
-        </para>
-      </listitem>
+      <glossentry>
+        <glossterm>Schließen</glossterm>
 
-      <listitem>
-        <para><emphasis role="bold">Fraction:</emphasis> This is the smallest traded unit of the commodity,
-          expressed as a fraction of a single nominal unit. This unit is used by &app; accounts as
-          the default fraction for trades in the commodity.
-        </para>
-      </listitem>
-    </itemizedlist>
+        <glossdef>
+          <para>Beendet den Wertpapier-Editor.
+          </para>
+        </glossdef>
+      </glossentry>
+    </glosslist>
 
     <sect2 id="tool-commodity">
-      <title>Adding or Editing a Commodity</title>
-
-      <para>To add a new commodity (stock, or mutual fund), click on <guilabel>Add</guilabel>, and enter the
-        details of the commodity into the <emphasis role="bold">New Security</emphasis> dialog box.
-        To edit an existing commodity, select the commodity from the Commodities list, click the
-        <guibutton>Edit</guibutton> button, and edit the <emphasis role="bold">Edit
-        Security</emphasis> dialog box. To remove a commodity, select the commodity and click the
-        <guibutton>Remove</guibutton> button.
-      </para>
+      <title>Wertpapier anlegen oder bearbeiten</title>
 
-      <para>The fields in the New/Edit Security screens are the same as defined for the Security Editor
-        <xref linkend="tool-security-edit"></xref>. Below these fields are the options for Online
-        Quotes.
+      <para>Der Wertpapier-Editor stellt ein weiteres Dialogfenster zu Verfügung in dem Sie Wertpapiere neu
+        anlegen oder auch bearbeiten können. Mit dem selben Dialog können Sie ebenfalls Währungen
+        bearbeiten. Nachfolgend wird Ihnen die Möglichkeiten beschrieben, die der Editor Ihnen
+        bietet.
       </para>
+
+      <table id="Tools-SecurityEditor-Dialog">
```

Comment:
Netter Versuch, zerhaut aber das Layout in verschiedenen Formaten.
Schau dir mal die Seite in PDF an. LOL

Wir kommen über die 800 px in Yelp, HTML

adjust_dpi.sh ist auch nur für einspaltiges Layout entwickelt.
Daher vorübergehend deaktiviert, aber wenn du mal wieder knobeln willst … ;-)